fission-web-server/README.md

```diff
@@ -52,11 +52,23 @@ To mimic the full "fission stack" for local development, you can use the include
    -  `docker compose exec dns-auth pdnsutil create-zone runfission.test`
    -  `docker compose exec dns-auth pdnsutil create-zone fissionuser.test`
    -  `docker compose exec dns-auth pdnsutil create-zone fissionapp.test`
-4. Point your local DNS resolver to localhost.
+4. Create some initial DNS records:
```

Comment:
With postgres being in docker compose, I believe you need something like the following somewhere in here after docker compose up.

```
docker compose exec postgres psql -U postgres
postgres=# create database web_api
postgres-# \q
```

Or with `createdb` instead.

Reply:
> With postgres being in docker compose, I believe you need something like the following somewhere in here after docker compose up.

If you're using the `env.example` from the repo it defines `POSTGRES_DB` - which will create the database on first run (see https://github.com/docker-library/docs/blob/master/postgres/README.md#postgres_db )